MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/ReflDataProcessorPresenter.h

```diff
@@ -4,6 +4,7 @@
 #include "MantidQtMantidWidgets/DataProcessorUI/GenericDataProcessorPresenter.h"
 #include "MantidQtMantidWidgets/DataProcessorUI/DataProcessorMainPresenter.h"
 #include "MantidQtMantidWidgets/DataProcessorUI/DataProcessorTreeManager.h"
+#include "MantidAPI/IEventWorkspace_fwd.h"
```

Comment:
It contains the alias `IEventWorkspace_sptr` which is used as the return value of `retrieveWorkspaceByName`. Which is a chunk of code I extracted from `parseUniform` to make it more readable.